frontend/src/views/Lobby.tsx

```diff
@@ -27,6 +26,9 @@ function LobbyPage() {
   // Variable to hold whether the user is connected to the socket.
   const [socketConnected, setSocketConnected] = useState(false);
 
+  // Dummy room id for dynamic room endpoint purposes
+  const socketRoomId = '791894';
```

Comment:
nit: Add `: string` for the type.

Reply:
How would we get the actual `socketRoomId`? I assume this be set in the `location` object, but I just wanted to confirm this.

Reply:
Yeah it will be passed using location (I do this in my next PR)